### PR TITLE
[Feature:US18225] Update license to be MIT

### PR DESCRIPTION
**Pull request by BreaGaudioso**
https://rally1.rallydev.com/#/24755394555d/detail/userstory/53778692751

### CODE REVIEW

**Comment by BreaGaudioso** [comment]
@serv Updated!

File: rhapsody.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2016 Rhapsody International
+// Code released under the MIT license.
+// See README for more detail.
```
Comment (serv) [inline rhapsody.js]: Let's give them the explicit URL for the readme.
https://github.com/Rhapsody/rhapsody.js#the-mit-license-mit
This way even if rhapsody.js is copied to someone else's repo, they can still find the original license.

**Comment by serv** [comment]
Check out how gulp build works.
https://github.com/Rhapsody/rhapsody.js/blob/master/gulpfile.js#L12

I think that when you run gulp build, it should also edit the minified files and other source files.

This will automatically add the license comment in other js files.

**Comment by BreaGaudioso** [comment]
* Added link to license section of README in comment
* Added the comment to the source file 
* Ran gulp build which added comment in areas specified in the task.

**Comment by serv** [comment]
Looks good. Merging.

**Pull request merged**